packages/nocms-plugin-directory-resources/src/renderDirectoryResource.js

```diff
@@ -1,20 +1,40 @@
 import renderDirectoryResourceCss from './renderDirectoryResourceCss';
 import renderDirectoryResourceHtml from './renderDirectoryResourceHtml';
+import renderDirectoryResourceJs from './renderDirectoryResourceJs';
 import cssBingo from 'css-bingo';
 
 export default async function renderDirectoryResource(directoryResource, resources) {
 	try {
-		let css = await renderDirectoryResourceCss(directoryResource);
-		let html = await renderDirectoryResourceHtml(directoryResource, resources);
-		let processedCss = cssBingo(css, html);
-		let htmlWithPurifiedCssEmbedded = embedCssInHtml(processedCss, html);
+		const css = await renderDirectoryResourceCss(directoryResource);
+		const html = await renderDirectoryResourceHtml(directoryResource, resources);
+		const processedCss = cssBingo(css, html);
+		let htmlWithAssets = embedCssInHtml(processedCss, html);
 
-		return htmlWithPurifiedCssEmbedded;
+		try {
+			const js = await renderDirectoryResourceJs(directoryResource);
+			if(js !== ''){
+				htmlWithAssets = embedJsInHtml(js, htmlWithAssets);
+			}
+		} catch(e) {
+			console.log('JS_ERROR');
```

Comment:
I am not sure if we should console.log.

Perhaps we should be doing a `debug` :)

Reply:
I thought about that, but bundling errors are syntax errors in the frontend JS, which will otherwise be hidden when writing frontend code in general